packages/ui/cypress/support/commands.ts

```diff
@@ -85,7 +85,7 @@ Cypress.Commands.add('rejectTx', (id: number, reason: string) => {
 
 Cypress.Commands.add('rejectCurrentMultisigTx', rejectCurrentMultisigTxs)
 
-Cypress.Commands.add('connectAccounts', (accountAddresses = [Account1] as string[]) => {
+Cypress.Commands.add('connectAccounts', (accountAddresses = [Account1] as string[], checkMultiProxyVisible: boolean = true) => {
```

Comment:
@Tbaut I had to add this and the if later in the block because if accounts aren't part of a multisig then there's no multiproxy dropdown. Let me know if you want to do it differently